docs/articles/hocon/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 uid: hocon
-title: Hocon module
+title: HOCON Syntax and Practices in Akka.NET
```

Comment:
@Arkatufus moved the HOCON file under articles/hocon/index.md - so maybe we can break that page up into several smaller chunks in the future or we can add more examples to that area.